### DISCUSSION TITLE
IDH - Installation small server

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#7620] [Unsupported Versions] [answered]
Hello,

Because IDH is only small services, any solution to install it on a small server and not a 100Go server? :/

Thank you.

**Comment by dougburks** [comment, marked as answer]
You could manually build your own honeypot, but be aware of the tradeoffs as discussed here:
https://github.com/Security-Onion-Solutions/securityonion/discussions/7522#discussioncomment-2341446

**Reply by b4b857f6ee** [reply]
Thank you.
